posthog/tasks/exports/image_exporter.py

```diff
@@ -87,7 +87,7 @@ def _export_to_png(exported_asset: ExportedAsset) -> None:
         wait_for_css_selector: CSSSelector
 
         if exported_asset.insight is not None:
-            url_to_render = absolute_uri(f"/exporter?token={access_token}&legend")
+            url_to_render = absolute_uri(f"/exporter?token={access_token}&legend&refresh=true&use_cache=true")
```

Comment:
Random observation: `refresh=if_stale` _could_ be clearer, because theoretically `refresh=false&use_cache=false` is now a legal combination – which should technically always return a cache miss